### ISSUE TITLE
crab version not properly reported in user-agent

### ISSUE THREAD

**Issue by belforte** [dmwm/CRABClient#4970]
crab client v3.210127 has been built in IB/pre but is still reports itself as CRABClient/0.0.0
at leat according to what I see in timber grafana dashboard.
OTOH the CLI reports correct version
```
belforte@lxplus757/TC3> crab --version
CRAB client v3.210127
belforte@lxplus757/TC3> 
```

**Comment by belforte** [comment]
I should have spotted this before, since when running from local GH I get
```
belforte@lxplus757/LOCAL> crab --version
CRAB client development
```
but CRABClient/0.0.0 in the header sent by pycurl.

**Comment by belforte** [comment]
there's a bug in CRABServer/RESTInteractions.py : https://github.com/dmwm/CRABServer/issues/6368
should be solved in new CRABServer tag[v3.210128](https://github.com/dmwm/CRABServer/releases/tag/v3.210128)
so made a new [build request](https://github.com/cms-sw/cmsdist/pull/6606)

**Comment by belforte** [comment]
solved. Current crab-dev in IB reports correctly
